src/main/java/com/brightpath/learnify/persistance/note/NoteEntity.java

```diff
@@ -61,4 +63,7 @@ public class NoteEntity {
     @OneToOne(fetch = FetchType.LAZY)
     @JoinColumn(name = "permissions_access_id")
     private PermissionsAccessEntity permissionsAccess;
+
+    @Embedded
```

Comment:
what is it doing?

Reply:
It wraps up additional columns (which are the same in the notes and quizzes) with embedded object. I used that to avoid duplication of logic. From the database perspective there is no additional table for this entity